Client/Cliqz/Frontend/Browser/Dashboard/TabsViewController.swift

```diff
@@ -158,11 +158,20 @@ extension TabsViewController: UITableViewDataSource, UITableViewDelegate {
 		cell.selectionStyle = .None
 		cell.isPrivateTabCell = tab.isPrivate
 		cell.animator.delegate = self
-		if let icon = tab.displayFavicon {
-			cell.logoImageView.sd_setImageWithURL(NSURL(string: icon.url)!)
-		} else {
-			cell.logoImageView.image = nil
-		}
+//		if let icon = tab.displayFavicon {
```

Comment:
Checking for indexpath is needed to support reusability of cells. Otherwise it'll cause another bug